hack/podman.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+cd $(dirname ${BASH_SOURCE[0]})
+
+action=$1
+
+if [ "$action" ]; then
+    [ -x "$action.sh" ] && action=hack/$action.sh
+else
+    histfile=.bash_history
+    [ -f $histfile ] || >$histfile
+
+    interactive="-it -v $PWD/$histfile:/root/$histfile"
+
+    dlv_conf=.dlv
+    [ -d $dlv_conf ] || mkdir $dlv_conf
+
+    interactive="$interactive -v $PWD/$dlv_conf:/root/.config/dlv"
+fi
+
+gopath_vol=${KAPP_GOPATH_VOL:-gopath}
+gobuild_vol=${KAPP_GOBUILD_VOL:-gobuild}
+
+# SYS_PTRACE for dlv
+# host network for minikube
+podman run --rm \
+    -v $gopath_vol:/go \
+    -v $gobuild_vol:/root/.cache/go-build \
+    -v $HOME/.kube:/root/.kube \
+    -v $HOME/.minikube:$HOME/.minikube \
+    -v $(realpath $PWD/..):/mnt -w /mnt \
```

Comment:
Maybe add a comment that realpath would require `brew install coreutils` on OSX?

Reply:
My podman scripts most likely are generally only compatible with Linux, not sure how the PWD mount and host network will work on OSX where Podman runs in a VM.